lib/services/apimanagement/lib/models/requestReportCollection.js

```diff
@@ -50,7 +50,7 @@ class RequestReportCollection extends Array {
                   }
               }
             }
-          },
+          }
```

Comment:
manually add the `,` for now or fix this in the generator /cc @daschult

Reply:
I'll fix the generator and then regenerate.

Reply:
Fixed in https://github.com/Azure/autorest.nodejs/pull/43.